src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs

```diff
@@ -24,10 +24,6 @@ public void RegisterDependencies(IIocContainer container)
             {
                 container.Register<IMailService, DebugMailService>(IocLifetimeEnum.Scoped);
             }
-            else if (Is<IISDeployment>.Enabled)
-            {
-                container.Register<IMailService, GmailAccountMailService>(IocLifetimeEnum.Scoped);
-            }
             else
             {
                 container.Register<IMailService, SendGridMailService>(IocLifetimeEnum.Scoped);
```

Comment:
This was not in AC but SendGrid should be active only when `Is<IISDeployment>.Disabled`

Reply:
http://www.innolution.com/resources/glossary/acceptance-criteria